chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -71,6 +72,10 @@ const PRUNE_UNREACHABLE_PEERS_AFTER: time::Duration = time::Duration::hours(1);
 /// Remove the edges that were created more that this duration ago.
 const PRUNE_EDGES_AFTER: time::Duration = time::Duration::minutes(30);
 
+/// How often should we refresh a nonce from a peer.
+/// It should be smaller than PRUNE_EDGES_AFTER.
+const REFRESH_NONCE_PERIOD: time::Duration = time::Duration::minutes(10);
```

Comment:
how about making it sth like
const REFRESH_NONCE_PERIOD = PRUNED_EDGES_AFTER/3

Reply:
done.